clients/fidesui/src/components/floating-menu/FloatingMenu.tsx

```diff
@@ -0,0 +1,14 @@
+import { AntMenu } from "fidesui";
```

Comment:
This one is a tricky question. Since we're currently in fidesui, do you think this should just import directly from Ant? That's what I ended up doing in the HOC. If not we should probably use "as" here as well, since we'll clean that up later.

Reply:
Yes! I've updated it to import from ant/lib directly.